### PR TITLE
flare throw fix

### LINKED ISSUE #14350: Flares no longer break resin nodes
### Description

You used to be able to break resin nodes by throwing flares at them. This no longer happens.

### Test Merges

Server-Side Code Modifications: https://tgstation13.org/parsed-logs/sybil/server_side_modifications.dm

Round ID: [24036](https://scrubby.melonmesa.com/round/24036)
BYOND Version: 515.1610
Server revision compiled on: 2023-10-31T21:47:21+00:00
Master commit: [6e076861aecba0b60fa227dae516961bdc151723](https://github.com/tgstation/TerraGov-Marine-Corps//commit/6e076861aecba0b60fa227dae516961bdc151723)
The following pull requests are currently test merged:
[#14258: 'Reverts Hyperscale 3' by lKiseki at commit 9667f76099](https://github.com/tgstation/TerraGov-Marine-Corps//pull/14258)
[#14298: 'Adds Library disk to Big Red, hardsets ETA disk.' by Barnet2 at commit 907e309a73](https://github.com/tgstation/TerraGov-Marine-Corps//pull/14298)
[#14262: 'Larva Queue: Now while playing Minions' by ZeWaka at commit 2f3163d432](https://github.com/tgstation/TerraGov-Marine-Corps//pull/14262)
[#14247: 'New hivelord primo and xeno flowers as primo for all builders' by Waselon at commit 7c325e0559](https://github.com/tgstation/TerraGov-Marine-Corps//pull/14247)

Local commit: 04996c9eb2da1bbc1ee0a9685b7a273159fde713
Server tools version: 5.16.4

### Reproduction Steps

1. Get flare
2. Light flare
3. Throw flare at resin node

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Fixes flares not extinguishing on nodes when you throw them.
Fixes #14350
## Why It's Good For The Game
Fix good.
## Changelog
:cl:
fix: fixed flares not hitting weed nodes
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
#14350

File: code/game/objects/items/explosives/grenades/marines.dm [hunk 1]
```diff
@@ -527,6 +527,11 @@
 		set_light_on(FALSE)
 
 /obj/item/explosive/grenade/flare/throw_impact(atom/hit_atom, speed)
+	if(isopenturf(hit_atom))
```
Comment (ivanmixo) [inline code/game/objects/items/explosives/grenades/marines.dm:530]: Don't see a case where a node would be outside an open turf?

**Review by ivanmixo — CHANGES_REQUESTED** [review]
Don't see a case where a node would be outside an open turf

File: code/game/objects/items/explosives/grenades/marines.dm (hunk 1 above)
Comment (Lumipharon) [reply code/game/objects/items/explosives/grenades/marines.dm:530]: needs to check for open turf as its happening before calling parent.

**Pull request merged**